airflow/api_fastapi/core_api/routes/public/dag_run.py

```diff
@@ -147,3 +152,48 @@ def patch_dag_run(
     dag_run = session.get(DagRun, dag_run.id)
 
     return DAGRunResponse.model_validate(dag_run, from_attributes=True)
+
+
+@dag_run_router.post("/{dag_run_id}/clear", responses=create_openapi_http_exception_doc([401, 403, 404]))
+def clear_dag_run(
+    dag_id: str,
+    dag_run_id: str,
+    patch_body: DAGRunClearBody,
```

Comment:
The route is a post, so `post_body` ? but actually just `body` is a better name I think. I started doing the `patch_body` but I don't think it's great.
```suggestion
    body: DAGRunClearBody,
```